tests/pkgconfig/sdk/pkgconfig.c

```diff
@@ -0,0 +1,6 @@
+#include <CL/cl.h>
+
+int main()
+{
+    return sizeof(cl_platform_id) - sizeof(cl_context);
+}
```

Comment:
This is the same file as ` tests/pkgconfig/bare/pkgconfig.c`. Do we need two copies?

Reply:
No, not really. It depends on what triggers someone: code duplication of a "one-liner" or referencing source files outside the project dir. You may be right, that code duplication can more easily bite. Fixed.

Reply:
Yeah, agree it was not a lot of duplication. Thanks for removing it anyway.